test/persistence-hooks.suite.js

```diff
@@ -575,6 +764,45 @@ module.exports = function(dataSource, should) {
           done();
         });
       });
+
+      describe('persist hook', function() {
+        
+
+        it('triggers `persist` hook on updateOrCreate', function(done) {
```

Comment:
Please move this test to the existing test suite `describe('PersistedModel.updateOrCreate')` per my earlier [comment](https://github.com/strongloop/loopback-datasource-juggler/pull/586/files#r31209708).